StudyHub/StudyHub.BLL/Services/OptionsService.cs

```diff
@@ -25,23 +25,11 @@ public class OptionsService : IOptionsService
         _mapper = mapper;
     }
 
-    public async Task<List<AssignmentTaskOptionDTO>> AddAssignmentTaskOptionsAsync(Guid assignmentTaskId, List<CreateAssignmentTaskOptionDTO> taskOptions)
-    {
-        // ToDo: remove unnecessary call
-        return await AddSeparationOptions(assignmentTaskId, taskOptions);
-    }
-
-    public async Task<List<AssignmentTaskOptionDTO>> UpdateAssignmentTaskOptionsAsync(Guid taskId, List<UpdateAssignmentTaskOptionDTO> dto)
-    {
-        // ToDo: remove unnecessary call
-        return await UpdateSeparationOptions(taskId, dto);
-    }
-
-    private async Task<List<AssignmentTaskOptionDTO>> UpdateSeparationOptions(
+    public async Task<List<AssignmentTaskOptionDTO>> UpdateAssignmentTaskOptionsAsync(
         Guid assignmentTaskId,
-        List<UpdateAssignmentTaskOptionDTO> taskOptions)
+        List<UpdateAssignmentTaskOptionDTO> dto)
```

Comment:
here list of dtos, but naming it dtos looks weird, so let's keep it like it was here